tools/src/tokenize.rs

```diff
@@ -102,7 +102,7 @@ pub fn tokenize_c_like(string: &String, spec: &LanguageSpec) -> Vec<Token> {
 
     while cur_pos.get() < chars.len() {
         let (start, ch) = get_char();
-        if is_ident(ch) {
+        if is_ident(ch) && (ch != 'R' || peek_char() != '"') {
```

Comment:
Could you reverse the order here so that the raw literal thing came first? Then you could test:
if ch == 'R' and peek_char() == '"'
And then the ident case would just be:
else if is_ident(ch)